### ISSUE TITLE
isShownAboveStatusBar

### ISSUE THREAD

**Issue by hhtopcu** [ephread/Instructions#127]
It's stated in the documentation that "The overlay can sit above the status bar or below, the property defaults to true:"

> overlay.isShownAboveStatusBar: Bool

But isShownAboveStatusBar is not defined in OverlayManager?

**Comment by ephread** [comment]
Hey @hhtopcu! It seems that the README is outdated. The property has been replaced by [`windowLevel`](https://github.com/ephread/Instructions/blob/master/Examples/Example/Source/Core/View%20Controllers/MainTableViewController.swift#L16), which is more flexible. I'll fix it.

**Comment by hhtopcu** [comment]
Thank you @ephread, after some trials, I understand that my problem is a little bit different. I've attached 2 screenshots. In the original one, the status bar is hidden. Whenever the coach marks are being displayed, the status bar becomes visible. I don't want the status bar displayed when coach marking is started. I just put a simple code to CoachMarksViewController below, It is started to be displayed as I expected. I think status bar visibility should be adjustable

```swift
override var prefersStatusBarHidden: Bool {
        return true
}
``` 

ORIGINAL VIEW (where status bar is not displayed)
![original](https://user-images.githubusercontent.com/4476368/28141286-ca955d28-6710-11e7-92d7-8c9a3352bbbb.jpg)

AFTER COACH MARKING (Status bar is visible, but I don't want this)
![coach marked](https://user-images.githubusercontent.com/4476368/28141285-ca785110-6710-11e7-94c4-47c83bbaeee9.jpg)

**Comment by ephread** [comment]
Very good point @hhtopcu, that's an oversight on my end. I'll make the `CoachMarkViewController` retrieve the value from the _parent_ VC (value passed to `start(on:)`) `prefersStatusBarHidden`, rather than exposing a way to define the value on the `internal CoachMarkViewController`. Does this sound good to you?

**Comment by hhtopcu** [comment]
Thank you @ephread, It sounds good 👍

**Comment by ephread** [comment]
Should be solved by 1.1.0. On start, Instructions will now retrieve `prefersStatusBarHidden` from the controller.
